docs/applications/voip/setup-a-mumble-server-on-debian.md

```diff
@@ -0,0 +1,154 @@
+---
+author:
+    name: Linode Community
+    email: docs@linode.com
+description: 'Setting up Mumble on Debian'
+keywords: 'mumble, debian, murmur, open source, gaming, VOIP, voice chat'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Thursday, September 24th, 2015'
+modified: Thursday, September 24th, 2015
+modified_by:
+    name: Linode
+title: 'Mumble on Debian'
+contributor:
+    name: Your Name
+    link: Github/Twitter Link
+---
+
+Mumble is an open-source VoIP client, designed for gamers, that requires a server for all clients to connect to. This guide instructs you on how to install and configure the Mumble server (also called Murmur) on Debian.
+
+No changes need to be made to the system beforehand if you follow this guide. 
+
+Througout this guide, all commands are run as root.  If you are not root user, you can prepend each command with "sudo." 
+
+## Server Side
+
+### Install and Simple Setup
+
+1. Since Murmur is in the official Debian repositories, you can just use apt-get to install it.  Be careful though, the package is `mumble-server` and not `murmur`. Enter the following command (remember to prepend with sudo if needed):
```

Comment:
I would kill off that last sentence. It's obvious that you need to run the command to do the needful.